core/assets/bundles/bundle.properties

```diff
@@ -1084,12 +1084,12 @@ tutorial.waves.mobile = The[lightgray] enemy[] approaches.\n\nDefend the core fo
 tutorial.launch = Once you reach a specific wave, you are able to[accent] launch the core[], leaving your defenses behind and[accent] obtaining all the resources in your core.[]\nThese obtained resources can then be used to research new technology.\n\n[accent]Press the launch button.
 
 item.copper.description = The most basic structural material. Used extensively in all types of blocks.
-item.lead.description = A basic starter material. Used extensively in electronics and liquid transportation blocks.
-item.metaglass.description = A super-tough glass compound. Extensively used for liquid distribution and storage.
+item.lead.description = A basic starter material. Used a lot in electronics and liquid transportation blocks.
```

Comment:
"a lot" and "often" don't sound right; you could use a [synonym](https://www.thesaurus.com/browse/extensive).

Reply:
Ok, I'll take a look.